RocketRL/python/main.py

```diff
@@ -25,19 +25,86 @@
 
 @author: ninalopatina
 """
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
 
 #import the functions from the functions file
 import func.data_processing as RocketData
-import func.run_env as RocketRunEnv
-import func.RL_results as RocketPlot
 
-#TO do: set up class and run like this:
-#rocketData = RocketData()
-#RocketData.plotter('2d')
+from gym.envs.RocketRL.RocketEnv_2T import TwoTemp
 
-#Load and process data:
-df, df_mini = RocketData.data_process(plotting2d = True)
+#import a few other funcs for the main script
+import argparse
+import os
+import yaml
 
-#Run your agent in the custom environment:
-df, df1, df_experiment = RocketRunEnv.run_env()
-RocketPlot.plot_results(df, df1, df_experiment)
+import gym
+from gym import spaces
+from gym.envs.registration import EnvSpec
+import pickle
+
+import ray
+from ray.tune import run_experiments
+from ray.tune.registry import register_env
+
+#to run: python main.py -run True -plot False
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-reg','--run_reg', dest = 'run_regression', type = int, 
+                        default = 2, help = '1 to run the regression, 2 to save it. 0 neither.')   
```

Comment:
How about adding a separate flag to save it? This seems a little bit confusing to me.

Reply:
good idea